### PR TITLE
Add Whatis Document(en)

### PR DESCRIPTION
**Pull request by ebbyzhang**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

Add Whatis Document(en)

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/09-英文/01-Introduction/03whatis.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+##  Broker
+
+Broker is mainly responsible for message storage, delivery and query, as well as service high availability guarantee.
+
+NameServer is stateless, so it can be deployed in clusters without any information synchronization between nodes.Compared with nameserver, broker is more complicated.
+
+In the Master-Slave architecture, the Broker is divided into Master and Slave.
+A Master can correspond to multiple Slaves, but a Slave can only correspond to one Master.
+The correspondence between Master and Slave is defined by specifying the same BrokerName and different BrokerId. A BrokerId of 0 means Master, and non-0 means Slave.Master can also deploy multiple.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:123]: > The correspondence between Master and Slave is defined by specifying the same BrokerName and different BrokerId. A BrokerId of 0 means Master, and non-0 means Slave.Master can also deploy multiple.

to:

> The correspondence between Master and Slave can be defined by specifying the same BrokerName and different BrokerId. A BrokerId 0 means Master, and non-0 means Slave. Master could be deployed more than one.

File: docs/09-英文/01-Introduction/03whatis.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+
+
+:::note RocketMQ Architecture Summary
+
+- Each **Broker** establishes long-term connections with all nodes in the **NameServer** cluster, and regularly registers Topic information to all NameServers.
+- **Producer** establishes a persistent connection with one of the nodes in the **NameServer** cluster, regularly obtains topic routing information from NameServer, establishes a persistent connection to the Master that provides Topic services, and regularly sends a heartbeat to the Master.Producers are completely stateless.
+- **Consumer** establishes a persistent connection to one of the nodes in the **NameServer** cluster
+，regularly obtains Topic routing information from NameServer,establishes long connections to Master and Slave that provide Topic services, and send heartbeats to Master and Slave regularly.Consumer subscribes to topic from Master or Slave.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:132]: > - **Consumer** establishes a persistent connection to one of the nodes in the **NameServer** cluster
，regularly obtains Topic routing information from NameServer,establishes long connections to Master and Slave that provide Topic services, and send heartbeats to Master and Slave regularly.Consumer subscribes to topic from Master or Slave.

to:

> - **Consumer** establishes a persistent connection to one of the nodes in the **NameServer** cluster, regularly obtains Topic routing information from NameServer, establishes long connections to Master and Slave that provide Topic services, and sends heartbeats to Master and Slave regularly. A Consumer can subscribe to Topics from either Master or Slave.

File: docs/09-英文/01-Introduction/03whatis.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+:::
+
+## RocketMQ Workflow
+
+### 1. Start the RocketMQ NameServer
+
+It listens to the port and waits for the connection of the Broker, Producer, and Consumer ater
+the NameServer starts.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:141]: > It listens to the port and waits for the connection of the Broker, Producer, and Consumer ater
the NameServer starts.

to:

> The NameServer listens to the port and waits for the connection of the Broker, Producer, and Consumer after startup.

File: docs/09-英文/01-Introduction/03whatis.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+### 2. Start the RocketMQ Broker
+
+It maintains long connections with all NameServers, gets current Broker information and store all Topic information when a broker start.After successful registration, there is a mapping relationship between Topic and Broker in the NameServer cluster.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:145]: > It maintains long connections with all NameServers, gets current Broker information and store all Topic information when a broker start.After successful registration, there is a mapping relationship between Topic and Broker in the NameServer cluster.

to:

> The Broker maintains long connections with all NameServers, gets current Broker information, and stores all Topic information after startup. After successful registration, a mapping relationship will be built between Topic and Broker in the NameServer cluster.

File: docs/09-英文/01-Introduction/03whatis.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+### 3. Create a topic
+
+Broker should be set up before creating a topic.And topic can be automatically created when sending a message.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:149]: > Broker should be set up before creating a topic.And topic can be automatically created when sending a message.

to:

> The Broker should be specified before creating a Topic, or automatically create one while sending messages.

File: docs/09-英文/01-Introduction/03whatis.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+### 4. Write messages to the topic
+
+The producer starts to establish a long connection with one of the groups in the NameServer, and obtains which Brokers the currently sent topic exists on from the NameServer, selects a queue from the queue list using polling, and then establishes a long connection with the first Broker to connect to. Broker message.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:153]: > The producer starts to establish a long connection with one of the groups in the NameServer, and obtains which Brokers the currently sent topic exists on from the NameServer, selects a queue from the queue list using polling, and then establishes a long connection with the first Broker to connect to. Broker message.

to: 

> The Producer starts by establishing a long-term connection with one device of the NameServer clusters, obtains the Broker information where the current topic exists from the NameServer, polls to select a queue from the queue list, and establishes a long-term connection where the queue is located. This enables the Producer to send messages to the Broker.

File: docs/09-英文/01-Introduction/03whatis.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+### 5. Read messages from the topic
+
+The consumer establishes a long connection with one of the NameServers, obtains which brokers the current subscription topic exists on, and then directly establishes a connection channel with the broker, and then starts to consume messages.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/01-Introduction/03whatis.md:157]: > The consumer establishes a long connection with one of the NameServers, obtains which brokers the current subscription topic exists on, and then directly establishes a connection channel with the broker, and then starts to consume messages.

to:

> The Consumer establishes a long-term connection with one of the NameServers, obtains which brokers the current subscription topic exists on, and then directly establishes a connection channel with the Broker, and then starts to consume messages.

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hi @ebbyzhang ,

Really appreciate your contribution. I've made some suggestions for you, it may seem a bit picky, but I hope this could make our documentation look more formal.

**Comment by tsunghanjacktsai** [comment]
Hi @ebbyzhang ,

Would you mind making some changes to this PR?

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**